### ISSUE TITLE
Auto completion for imports

### ISSUE THREAD

**Issue by sgronblo** [sbrink/vscode-elm#98]
Currently there seems to be no auto completion for imports

```
import Deb<cursor, press ctrl+space>
```

No suggestions shown. I would expect core's Debug to show up.

**Comment by witoldsz** [comment]
That would be nice to have, but what I miss the most is the ability to just write a function name somewhere in a code (or just part of that name) and let the editor suggest me to add the import by itself.

For example, in Netbeans or Idea, you type "format", editor marks an error and suggestion to import one of the "format" functions available. You can also type "form[ctrl+space]" and it will suggest different versions of format together with their package names.

That would be awesome :)

**Comment by sgronblo** [comment]
Yes it would be great. But that is a different feature so should probably be tracked in a separate issue.
